core/state_transition.go

```diff
@@ -287,8 +287,8 @@ func (st *StateTransition) preCheck() error {
 				st.msg.From().Hex(), codeHash)
 		}
 	}
-	// Make sure that transaction gasFeeCap is greater than the baseFee (post london)
-	if st.evm.ChainConfig().IsLondon(st.evm.Context.BlockNumber) {
+	// Make sure that transaction gasFeeCap is greater than the baseFee (post Banach)
+	if st.evm.ChainConfig().IsBanach(st.evm.Context.BlockNumber) {
```

Comment:
Behavior after London and before Banach:

- Original behavior: Step into `if` branch, execute checks.
- New behavior: Skip checks.

This might diverge, although I'm not sure how these fields would be set to longer than 256 bits. We can change back to `IsLondon` to be safe.

Reply:
Fixed in [9f9a069](https://github.com/scroll-tech/go-ethereum/pull/634/commits/9f9a0692001385ef74b852d9aae89d41f546cfc3).